app/src/main/java/eu/kanade/tachiyomi/data/backup/restore/restorers/MangaRestorer.kt

```diff
@@ -202,6 +202,7 @@ class MangaRestorer(
                 bookmark = chapter.bookmark || dbChapter.bookmark,
                 read = chapter.read,
                 lastPageRead = chapter.lastPageRead,
+                sourceOrder = chapter.sourceOrder
```

Comment:
If I understand correctly, the build failed because a comma should be at the end here?

Reply:
Seems to have been it